### ISSUE TITLE
Cannot updateMany Dynamic Content Variants

### ISSUE THREAD

**Issue by alarocca130** [zendesk/zendesk_api_client_php#409]
**TL;DR**

I think that `$objectName` and `$objectNamePlural` constants in `DynamicContentItemVariants` class should be changed to `'variant'` and `'variants'` values respectively

**Issue**
If you try to use updateMany with dynamic content variants, you always get the following error from server:

`{"error":"UnknownAttributeError","description":"Invalid attribute: missing variants parameter"}`

After some debugging, I find out that this is what updateMany is sending to the server as body of the PUT request

```
{
    "items": [
        {
            "id": 23,
            "content": "Lorem Ipsum",
        }
    ]
}
```

While official documentation suggest a body formatted as follows

```
{
  "variants": [
        {
          "id": 23,
          "content": "Lorem Ipsum",
        },
    ]
}
```

As you can see (API/Traits/Resource/UpdateMany.php:35), updateMany uses class constant value as resource name, so I suppose that this is the issue

**How to Replicate**
Here my test code:

```
$client->dynamicContent()->items( $dc_id )->variants()->updateMany( [
    [
        'id' => $v_id,
        'content' => 'Lorm Ipsum'
    ]
] );
```

where `$dc_id` contains id of the Dynamic content field that I want to update and `$v_id` contains id of the specific variation that I need to change

**Possible Solution**
Changing constants values with suggested ones seems to work pretty fine for me

**Note**
I'm using version 2.2.7 of the library, but I can see that `DynamicContentItemVariants` class wasn't changed since this version

I would be happy to repeat my test with latest version if you think it could help

**Comment by tzimpel** [comment]
What does it take to get this issue solved? Can you merge the corresponding PR, or is there some issue I could help resolve?

**Comment by AlexFalappa** [comment]
Please consider merging the referred PR, we are currently working around this by issuing lots of create single variant API calls which is clearly uneffective

**Comment by AlexFalappa** [comment]
Great!

Looking forward to a new release!
